inference-engine/src/mkldnn_plugin/utils/node_dumper.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#ifdef CPU_DEBUG_CAPS
+
+#include "node_dumper.h"
+
+#include "mkldnn_node.h"
+#include "utils/blob_dump.h"
+
+#include "ie_common.h"
+#include <array>
+#include <regex>
+#include <sstream>
+#include <string>
+
+using namespace InferenceEngine;
+
+namespace MKLDNNPlugin {
+
+void NodeDumper::setup() {
+    char* dumpDirEnv = getenv("OV_CPU_BLOB_DUMP_DIR");
+    if (dumpDirEnv)
+        dumpDirName = dumpDirEnv;
+
+    char* dumpAsTextEnv = getenv("OV_CPU_BLOB_DUMP_AS_TEXT");
```

Comment:
Maybe OV_CPU_BLOB_DUMP_AS_TEXT -> OV_CPU_BLOB_DUMP_MODE (possible values: bin, text, ...) to make extensible?

Reply:
will be updated

Reply:
Done